lib/experimental/class-wp-navigation-fallbacks-gutenberg.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * WP_Navigation_Fallbacks_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+
+require __DIR__ . '/class-wp-menu-conversion-gutenberg.php';
+
+/**
+ * Manages Fallback behavior for Navigation menus.
+ *
+ * @access public
+ */
+class WP_Navigation_Fallbacks_Gutenberg {
```

Comment:
Currently much of the code here is static.

Question: would it be preferable to make this into a "instance" based class?

Reply:
I don't see any benefit if the functions are static myself.

Reply:
> I don't see any benefit if the functions are static myself.

These functions do not require an object of the `WP_Navigation_Fallbacks_Gutenberg` class to operate. I am not sure of the benefit of making them non-static.

Reply:
These functions do not require an object of the `WP_Navigation_Fallbacks_Gutenberg` class to operate. I am not sure of the benefit of making them non-static.